packages/design-system-scripts/gulp/sass.js

```diff
@@ -55,16 +55,6 @@ function compileSass(src, dest, includePaths, browserSync) {
     )
     .pipe(gulp.dest(dest));
 
-  if (browserSync) {
```

Comment:
Nice! I think that means we can also remove it from the `compileSass` function and also remove the `browser-sync` dependency. It looks like this wasn't even hooked up before 😕

Reply:
Ahh nice, removed! That took a bunch of junk with it, nice.